test/run-focus

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Maybe want to `.gitignore` this? `git rm test/run-focus` `echo test/run-focs >> .gitignore` ?